hazelcast.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Hazelcast - a streaming and memory-first application platform
+After=syslog.target network.target
+
+[Service]
+SuccessExitStatus=143
+
+Type=simple
+
+ExecStart=hz-start
+ExecStop=/bin/kill -15 $MAINPID
```

Comment:
why not `hz-stop`?

Reply:
`hz-stop` kills all hz processes, in the unlikely case when you run this on your local and run another instance for some reason it would kill that as well. Moreover I think this is the standard way in systemd for `Type=simple`